src/neuroconv/datainterfaces/ophys/inscopix/inscopixsegmentationdatainterface.py

```diff
@@ -0,0 +1,251 @@
+
+            if device_desc_parts:
+                device_metadata["description"] = "; ".join(device_desc_parts)
+
+            # Update imaging plane metadata
+            imaging_plane_metadata = metadata["Ophys"]["ImagingPlane"][0]
+
+            # Update imaging plane device reference to match the actual device name
+            if device_info.get("device_name"):
+                imaging_plane_metadata["device"] = device_info["device_name"]
+
+            # Build imaging plane description
+            plane_desc = "Inscopix imaging plane"
+            if device_info.get("field_of_view_pixels"):
+                fov = device_info["field_of_view_pixels"]
+                plane_desc += f" with field of view {fov[1]}x{fov[0]} pixels"
+
+            desc_parts = [plane_desc]
+            if device_info.get("microscope_focus"):
+                desc_parts.append(f"Focus: {device_info['microscope_focus']} µm")
+            if device_info.get("exposure_time_ms"):
+                desc_parts.append(f"Exposure: {device_info['exposure_time_ms']} ms")
+            if device_info.get("microscope_gain"):
+                desc_parts.append(f"Gain: {device_info['microscope_gain']}")
+
+            imaging_plane_metadata["description"] = "; ".join(desc_parts)
+
+            # Sampling frequency
+            sampling_frequency = extractor.get_sampling_frequency()
+            if sampling_frequency:
+                imaging_plane_metadata["imaging_rate"] = sampling_frequency
+
+            # Optical channel
+            optical_channel = {
+                "name": "OpticalChannelDefault",
+                "description": "Inscopix optical channel",
+                "emission_lambda": float("nan"),
+            }
+            if device_info.get("channel"):
+                channel = device_info["channel"]
+                optical_channel["name"] = f"OpticalChannel{channel.capitalize()}"
+                optical_channel["description"] = f"Inscopix {channel} channel"
+
+            # LED power (segmentation uses different field name)
+            if device_info.get("led_power_1_mw_per_mm2"):
+                led_power = device_info["led_power_1_mw_per_mm2"]
+                optical_channel["description"] += f" (LED power: {led_power} mW/mm²)"
+
+            imaging_plane_metadata["optical_channel"] = [optical_channel]
+
+        # Image segmentation (specific to segmentation interface)
+        segmentation_desc = "Inscopix cell segmentation"
+        if analysis_info and analysis_info.get("cell_identification_method"):
+            segmentation_desc += f" using {analysis_info['cell_identification_method']}"
+        if analysis_info and analysis_info.get("trace_units"):
+            segmentation_desc += f" with traces in {analysis_info['trace_units']}"
+
+        # Add number of ROIs to description
+        num_rois = extractor.get_num_rois()
+        segmentation_desc += f" ({num_rois} ROIs identified)"
```

Comment:
No need to write this in the description. This is available to the user as the number of the rows in the plane segmentation.